src/vs/editor/contrib/codeAction/browser/lightBulbWidget.ts

```diff
@@ -224,6 +224,13 @@ export class LightBulbWidget extends Disposable implements IContentWidget {
 
 	private set state(value) {
 		this._state = value;
+		if (value.type === LightBulbState.Type.Showing) {
+			const onlyAIActions = value.actions.allAIFixes;
+			const showAiIcon = this._editor.getOption(EditorOption.lightbulb).experimental.showAiIcon;
+			if (onlyAIActions && showAiIcon === ShowAiIconMode.Off) {
```

Comment:
Now `LightBulbWidget.state,type` is `LightBulbState.Type.Showing`, but it's not showing. Or maybe it is? We actually don't know, as all the change does is to not call `this._updateLightBulbTitleAndIcon();`